src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,17 +156,20 @@ public RobotContainer() {
    * joysticks}.
    */
   private void configureBindings() {
-    OI.getButton(OI.Operator.A)
+    OI.getButton(OI.Driver.modeChangeButton) // Bound to X][\]
         .onTrue(
             new InstantCommand(robotStateManager::switchPlacementMode)
                 .withName("Switch Placement Mode Command"));
     if (Constants.enabledSubsystems.intakeEnabled) {
       OI.getTrigger(OI.Driver.intakeTrigger)
           .whileTrue(
-              intakeSubsystem
-                  .getIntakeCommand(robotStateManager.getPlacementMode())
-                  .withName("Get Placement Mode Command"));
-      OI.getButton(OI.Driver.outtakeButton)
+              Commands.either(
+                  intakeSubsystem.getSpeakerIntakeCommand(),
+                  Commands.parallel(
```

Comment:
Should we have a `.withName()` on this command?

Reply:
the commands inside have a withName. Is it still useful to name it?